### ISSUE TITLE
[Bug]: organize_files_tops_linux.csh creates faulty XML files

### ISSUE THREAD

**Issue by ericlindsey** [gmtsar/gmtsar#849]
**Describe the bug**
When running organize_files_tops_linux.csh in mode 2, re-framed TIFF files are written out but the associated XML files have a subtle bug in the last few lines. The original last lines of the XML files are as follows:

`  <coordinateConversion>
    <coordinateConversionList count="0"/>
  </coordinateConversion>
  <swathMerging>
    <swathMergeList count="0"/>
  </swathMerging>
</product>`

However, the new XML files are as follows:

`  <coordinateConversion>
    <coordinateConversionList count="0"/></coordinateConversionList>
  </coordinateConversion>
  <swathMerging>
    <swathMergeList count="0"/></swathMergeList>
  </swathMerging>
</product>`

The issue is that the tags `coordinateConversionList` and `swathMergeList` are closed twice - the ending "/" inside the initial tag already closes the tag, so adding the explicit </ > to close it is unnecessary.

This bug doesn't seem to affect the running of GMTSAR, but it prevents using the files generated via this process within ISCE, which has a lower tolerance for bugs in the XML tags. I think this is worth fixing because it should be a relatively minor change.

**To Reproduce**
Run `organize_files_tops_linux.csh` with some data, and inspect the resulting XML files in the created SAFE folders.

**Comment by Xiaohua-Eric-Xu** [comment]
Yup, this is a bug associated with function `int print_tree(struct tree *T, int ct, int mode, FILE *fp)` in `xml.c`. 

I made a fix here (e0bd2fe837be076a5bc1bacc72dc96a9682aaf62) but does not have a ready example to test it out. Do you mind giving us a hand?
